src/js/media-text-display.js

```diff
@@ -42,9 +42,7 @@ template.innerHTML = `
       outline: 0;
     }
   </style>
-  <span id="container">
```

Comment:
question(non-blocking): Were there no ripple effects on styling when removing this? Any concerns over how this may impact subclasses ability to specify roles and related a11y attributes without sprouting?

Reply:
no styling issues that I saw. it's possible to set any HTML in the slot so should be fine